src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js

```diff
@@ -138,45 +349,30 @@ function showDatasets(by,id) {
                             }
                         }
                     }
+                    var activeTabIndex = getActiveTabIndex();
+                    currentIndex = (datasetList[activeTabIndex].length === 0) ? 0 : datasetList[activeTabIndex].length - 1;
```

Comment:
Is this check required? because currentIndex and datasetList values are reset in this same function

Reply:
Yes because the indices are different for every tab. This code is in the activate function which will triggers when switching the tabs.

Reply:
I think the use of global variables is not encouraged..  it will become confusing for us, when we look at the code sometime later